docs/tutorial/tutorial-0.rst

```diff
@@ -75,6 +76,8 @@ Next, install Toga into your virtual environment:
 
       (venv) C:\...>pip install --pre toga
 
+If you get other errors, please check that you followed `prerequisites <https://github.com/beeware/toga#prerequisites>`_ instructions.
```

Comment:
One more update: please check that you followed *the* *prerequisite* instructions.

Reply:
It is not *the prerequisite* (without the **s**)?

Reply:
fixed: https://github.com/roipoussiere/toga/commit/b36a8bfadc7a63248c9c53cf025c5d5540ee55e0